cirq-superstaq/cirq_superstaq/circuits/msd_15_to_1_test.py

```diff
@@ -0,0 +1,43 @@
+import cirq
+import numpy
+from MSD_15_to_1 import msd_15_to_1
+
+
+def test_consistent_state() -> None:

```

Comment:
Checking the state vector was well and good for help writing the code, but we don't need to have it as a test because it tests the same thing as `test_similarities`. We can remove it.